substrate/frame/im-online/Cargo.toml

```diff
@@ -23,6 +23,7 @@ frame-benchmarking = { path = "../benchmarking", default-features = false, optio
 frame-support = { path = "../support", default-features = false }
 frame-system = { path = "../system", default-features = false }
 pallet-authorship = { path = "../authorship", default-features = false }
+pallet-session = { path = "../session", default-features = false }
```

Comment:
There https://github.com/paritytech/polkadot-sdk/blob/654cd7214273da94c7a8a6934c525a6481eb7b76/substrate/frame/im-online/src/lib.rs#L281
type that can be used here, no need to add a dependency on pallet-session.

Reply:
Hmmm, not sure how it is supposed to work. Yes, we have the type, but we're getting the actual validator set from pallet-session to find its size. And im-online on-chain storage has already been removed by the time when this code runs, so we cannot get validator set size from im-online itself.